src/pages/payment-methods/_expanded-list.js

```diff
@@ -91,12 +91,15 @@ const Withdrawal = styled(Td)`
     }
 `
 
-const ExpandList = ({ data }) => {
+const ExpandList = ({ data, config, is_crypto }) => {
     const [is_expanded, setIsExpanded] = React.useState(false)
 
     const toggleExpand = () => {
         setIsExpanded(!is_expanded)
     }
+    function getCryptoConfig(name) {
```

Comment:
can we standardize this to use arrow function

Reply:
done :+1: